[2023-04-28T21:55:08.553+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv4.exec_insert_query_task manual__2023-04-28T21:54:59.793198+00:00 [queued]>
[2023-04-28T21:55:08.564+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv4.exec_insert_query_task manual__2023-04-28T21:54:59.793198+00:00 [queued]>
[2023-04-28T21:55:08.565+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-04-28T21:55:08.565+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-04-28T21:55:08.565+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-04-28T21:55:08.584+0000] {taskinstance.py:1303} INFO - Executing <Task(PostgresOperator): exec_insert_query_task> on 2023-04-28 21:54:59.793198+00:00
[2023-04-28T21:55:08.592+0000] {standard_task_runner.py:55} INFO - Started process 1255 to run task
[2023-04-28T21:55:08.599+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv4', 'exec_insert_query_task', 'manual__2023-04-28T21:54:59.793198+00:00', '--job-id', '451', '--raw', '--subdir', 'DAGS_FOLDER/todays_games.py', '--cfg-path', '/tmp/tmphr4jnbyw']
[2023-04-28T21:55:08.600+0000] {standard_task_runner.py:83} INFO - Job 451: Subtask exec_insert_query_task
[2023-04-28T21:55:08.680+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv4.exec_insert_query_task manual__2023-04-28T21:54:59.793198+00:00 [running]> on host f789e77bc70a
[2023-04-28T21:55:08.797+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv4
AIRFLOW_CTX_TASK_ID=exec_insert_query_task
AIRFLOW_CTX_EXECUTION_DATE=2023-04-28T21:54:59.793198+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-28T21:54:59.793198+00:00
[2023-04-28T21:55:08.798+0000] {sql.py:254} INFO - Executing: 
INSERT INTO games (away_name,home_name,away_probable_pitcher,home_probable_pitcher,venue_name)
                                   VALUES ('New York Yankees','Texas Rangers','Clarke Schmidt','Jacob deGrom','Globe Life Field');
[2023-04-28T21:55:08.811+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-04-28T21:55:09.104+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-04-28T21:55:09.108+0000] {sql.py:375} INFO - Running statement: 
INSERT INTO games (away_name,home_name,away_probable_pitcher,home_probable_pitcher,venue_name)
                                   VALUES ('New York Yankees','Texas Rangers','Clarke Schmidt','Jacob deGrom','Globe Life Field');, parameters: None
[2023-04-28T21:55:09.109+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-28T21:55:09.121+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=get_todays_gamesv4, task_id=exec_insert_query_task, execution_date=20230428T215459, start_date=20230428T215508, end_date=20230428T215509
[2023-04-28T21:55:09.170+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-28T21:55:09.206+0000] {taskinstance.py:2585} INFO - 2 downstream tasks scheduled from follow-on schedule check
